src/components/picker.tsx

```diff
@@ -28,7 +28,7 @@ interface State {
   picked: ID[]
   mode: 'gallery' | 'table'
   search: string
-  page: number
+  sort: keyof Record
```

Comment:
I had to go look at Record to see what you were doing here.  I'd normally expect to see an Enum for a case like this.  It might be worth, given how often you are using `keyof Record` exporting a type alias for `keyof Record` so that could could have it read `sort: SortableColumn` or something like that.

Reply:
👍 Great idea. Done in fc41e1d